conf/base.config

```diff
@@ -104,13 +104,22 @@ process {
         memory = { 20.GB * task.attempt }
         time   = { 12.h * task.attempt }
     }
-    withName: CAT_DB_GENERATE {
+    withName: CATPACK_DOWNLOAD {
+        time   = { 12.h * task.attempt }
+    }
+    withName: CATPACK_PREPARE {
+        cpus   = { 10 * task.attempt }
         memory = { 200.GB * task.attempt }
         time   = { 16.h * task.attempt }
     }
-    withName: CAT {
-        cpus   = { 8 * task.attempt }
-        memory = { 40.GB * task.attempt }
+    withName: CATPACK_BINS {
+        cpus   = { 10 * task.attempt }
+        memory = { 120.GB * task.attempt }
+        time   = { 12.h * task.attempt }
+    }
```

Comment:
I was wondering why the default labels such as `process_high` are not used and found that `CATPACK_BINS` has `process_medium`. I would guess (but do not know) that CATPACK needs a lot of resources typically, so shouldn't it rather be `process_high` & `process_high_memory` in the nf-core module? Ofc for another day ;)
There is an extreme number of non-default label resources, why is that? Because the modules are so well optimized or because the nf-core modules have non-optimal labels?

Reply:
To be honest, I didn’t fine-tune this very carefully. I tested it with three samples, and in some cases, catpack bins used up to 113 GB of memory with 10 cores, which is why I set the limit to 120 GB. Regarding CPU and time, the custom settings are probably unnecessary, we can leave the defaults from the process label.

For the contigs, I simply copied the same configuration from the bins step. But now that I’ve looked at the actual resource usage, it seems to be using much less memory, up to 38 GB.

I’ll run a few more samples, check the resource usage and try to reduce the custom resource directives.

Reply:
I ran a few more samples using the default 'medium' resource label and observed the following:
- For catpack/bins, memory usage was consistently around 115 GB of RAM, with runtimes ranging from 2 to 6.5 hours.
- For catpack/contigs, memory usage was either under 40 GB or roughly the same as for bins (~115 GB), with runtimes between 1.5 and 3 hours.

Based on this, I removed most of the custom resource settings and only left memory: `120 GB * task.attempt` for catpack/bins and `60 GB * task.attempt` for catpack/contigs.